mobile-widgets/qml/DownloadFromDiveComputer.qml

```diff
@@ -179,7 +179,7 @@ Kirigami.Page {
 			Layout.alignment: Qt.AlignTop
 			Layout.topMargin: Kirigami.Units.smallSpacing * 2
 			spacing: Kirigami.Units.smallSpacing;
-			width: subsurfaceTheme.columnWidth - Kirigami.Units.gridUnit * 4
+			Layout.fillWidth: true
```

Comment:
when testing on Desktop there are situations where the drop down comboboxes extended to the right of the screen. By making this somewhat narrower I was able to keep the buttons fully visible.
But I basically agree that this is broken and that if this can happen at all, THAT needs to be fixed.

Reply:
What I did was to add a button with the same width as you had specified, that button used ~75% of the width of my 5" phone I use for testing this. 
Unless the "Layout.fillWidth: true" is set objects that should fit side by side wrap.

Reply:
Maybe that's another Android vs Desktop issue? Because smaller buttons would happily sit next to each other and a larger one would then stay by itself. So as written this worked as one would expect it. But I am OK with your change if this works on your phone.
I'm very frustrated that the scale thing doesn't work for you, though :-(